models/model.js

```diff
@@ -3,3 +3,11 @@ const db = require("../db/connection");
 exports.fetchCategories = () => {
   return db.query("SELECT * FROM categories").then((result) => result.rows);
 };
+
+exports.fetchReviews = () => {
+  return db
+    .query(
+      "SELECT reviews.*, COUNT(comments.comment_id)::INT AS comment_count FROM REVIEWS LEFT JOIN comments ON comments.review_id = reviews.review_id GROUP BY reviews.review_id ORDER BY created_at DESC"
```

Comment:
I would just like you to lowercase `reviews` here in the FROM clause, just so that its more obvious that this is a table. SQL commands in CAPS, values/tables in lowercase in future